generate-release-vars.sh

```diff
@@ -15,80 +16,79 @@ error() {
     exit 1
 }
 
-readonly ecs_init_version="$1"
-if [ -z "$ecs_init_version" ]; then
-    error "ecs-init version is required."
-fi
-readonly ami_version="$2"
-if [ -z "$ami_version" ]; then
-    error "ami version is required."
+readonly ami_type="$1"
+if [ -z "$ami_type" ]; then
+    error "AMI_TYPE must be provided"
 fi
 
-agent_version=$(echo "$ecs_init_version" | awk -F "-" '{ print $1 }')
-ecs_init_rev=$(echo "$ecs_init_version" | awk -F "-" '{ print $2 }')
-readonly agent_version ecs_init_rev
-if [ -z "$ecs_init_rev" ]; then
-    error "ecs-init rev was empty, did you forget the dash in ECS_INIT_VERSION? ie, 1.55.0-1"
-fi
-if [ -z "$agent_version" ]; then
-    error "agent version was empty, seems that your ECS_INIT_VERSION was malformed, it should look like: 1.55.0-1"
-fi
+ami_version=$(date +"%Y%m%d")
```

Comment:
nit. should we make `ami_version` readonly as well?

Reply:
Yea, although none of the readonly's are _that_ necessary, it would be less error prone in that case. I'll do that now.